diff --git a/breakout/tf_estimator/train_agent.py b/breakout/tf_estimator/train_agent.py
index 9860d94..5190dd9 100644
--- a/breakout/tf_estimator/train_agent.py
+++ b/breakout/tf_estimator/train_agent.py
@@ -27,8 +27,15 @@ if __name__ == '__main__':
     parser.add_argument('--output-dir', type=str, default='/tmp/training/')
     parser.add_argument('--log-time', default=False, action='store_true')
     parser.add_argument('--num-gpus', type=int, default=0)
+    parser.add_argument('--wandb', type=str, default=None)
     args = parser.parse_args()
 
+    if args.wandb is not None:
+        import wandb
+        wandb.init(name=args.wandb,
+                   project="rab-bigtable-rl",
+                   entity="42 Robolab")
+
     #INSTANTIATE CBT TABLE AND GCS BUCKET
     credentials = service_account.Credentials.from_service_account_file(SERVICE_ACCOUNT_FILE, scopes=SCOPES)
     cbt_table, gcs_bucket = gcp_load_pipeline(args.gcp_project_id, args.cbt_instance_id, args.cbt_table_name, args.bucket_id, credentials)
